deploy/arq/.env

```diff
@@ -4,4 +4,11 @@
 # and https://github.com/i-VRESSE/haddock3-webapp/blob/main/docs/auth.md#social-logins
 # respectively for more information.
 
-HADDOCK3WEBAPP_REFRESH_RATE_MS=30000
\ No newline at end of file
+HADDOCK3WEBAPP_REFRESH_RATE_MS=30000
+
+# GITHUB LOGIN
+# Provide clientId and secret obtained from github settings
+# HADDOCK3WEBAPP_GITHUB_CLIENT_ID=<GITHUB-CLIENT-ID-HERE>
+# HADDOCK3WEBAPP_GITHUB_CLIENT_SECRET=<GITHUB-CLIENT-SECRET-HERE>
+# Callback url consist of webapp domain + /auth/github/callback
+# HADDOCK3WEBAPP_GITHUB_CALLBACK_URL=<{HADDOCK3-WEBAPP-DOMAIN}/auth/github/callback>
```

Comment:
As the arg deployment is running on http://localhost:8080 . I might be nice to have `http://localhost:8080/auth/github/callback` as an example when you dont reverse proxy the webapp.